zeroex/orderwatch/order_watcher.go

```diff
@@ -73,7 +73,7 @@ type Watcher struct {
 	contractAddresses          ethereum.ContractAddresses
 	orderFeed                  event.Feed
 	orderScope                 event.SubscriptionScope // Subscription scope tracking current live listeners
-	contractAddressToSeenCount map[common.Address]uint
+	contractAddressToSeenCount *contractAddressesSeenCounter
```

Comment:
we can rename this field to `contractAddressSeenCounter` now that it is no longer a map.